libcontainer/seccomp/types.go

```diff
@@ -0,0 +1,48 @@
+package seccomp
```

Comment:
I don't think this API is as good as what we currently have.  The current model is more clear and you have the ability to set the exit status that you want returned and not having alot of strings for the syscalls and operators.  Those are both type safe currently.  Can you not use the existing API and just swap out the implementation with libseccomp?